chapters/raspberrypi.md

```diff
@@ -0,0 +1,154 @@
+Profile: Raspberry Pi / Raspberry Pi Foundation
+==================
+
+### Authors
+
+- Chris Knepper	[chris82thekid@gmail.com](mailto:chris82thekid@gmail.com)
+- Joe Trotta [42istheanswer.whatsthequestion@gmail.com](mailto:42istheanswer.whatsthequestion@gmail.com)
+- Derin Yarsuvat [misterberu@gmail.com](mailto:misterberu@gmail.com)
+
+### Rationale
+
+There are **so** many subjects you could choose to profile, so why did you choose this one?
+What drew you into wanting to know more about the organization?
+How did you/your group decide on and agree?
+
+The Raspberry Pi is a well-known open hardware device, used in a variety of contexts. It helps kids learn programming, is used as the brain of many DIY/physical computing projects, and because it is so cheap, serves to help people get a computer who otherwise might not be able to afford one. The organization responsible for the device is a registered charity.
+
+
+## Organizational Details
+
+- Is the subject of your profile a corporate entity?
```

Comment:
Q&A doesn't make much sense as a nested list. Markdown has blockquote support that would display better. 

> Is the subject of your profile a corporate entity

Yes

> What type?

blah blah blah